### PR TITLE
Fix broken bench scripts

### PR DESCRIPTION
**Pull request by mcollina**
Our bench CI job is not working anymore. This PR partially fixes it _but_ the number of iterations do not work anymore.

```
[bench:run] │ Tests               │ Samples │          Result │ Tolerance │ Difference with slowest │
[bench:run] |─────────────────────|─────────|─────────────────|───────────|─────────────────────────|
[bench:run] │ undici - fetch      │       1 │  596.19 req/sec │  ± 0.00 % │                       - │
[bench:run] |─────────────────────|─────────|─────────────────|───────────|─────────────────────────|
[bench:run] │ http - no keepalive │       1 │ 1123.28 req/sec │  ± 0.00 % │               + 88.41 % │
[bench:run] |─────────────────────|─────────|─────────────────|───────────|─────────────────────────|
[bench:run] │ http - keepalive    │       1 │ 1943.09 req/sec │  ± 0.00 % │              + 225.92 % │
[bench:run] |─────────────────────|─────────|─────────────────|───────────|─────────────────────────|
[bench:run] │ undici - request    │       1 │ 2084.66 req/sec │  ± 0.00 % │              + 249.66 % │
[bench:run] |─────────────────────|─────────|─────────────────|───────────|─────────────────────────|
[bench:run] │ undici - pipeline   │       1 │ 2142.13 req/sec │  ± 0.00 % │              + 259.30 % │
[bench:run] |─────────────────────|─────────|─────────────────|───────────|─────────────────────────|
[bench:run] │ undici - stream     │       1 │ 2847.98 req/sec │  ± 0.00 % │              + 377.69 % │
[bench:run] |─────────────────────|─────────|─────────────────|───────────|─────────────────────────|
[bench:run] │ undici - dispatch   │       1 │ 2941.40 req/sec │  ± 0.00 % │              + 393.36 % │
```

### CODE REVIEW

**Comment by mcollina** [comment]
@ShogunPanda @RafaelGSS could you take a look?

File: benchmarks/benchmark.js [hunk 1]
```diff
@@ -129,6 +129,8 @@ function printResults (results) {
       ]
     })
 
+  console.log(results)
```
Comment (RafaelGSS) [inline benchmarks/benchmark.js:132]:
```suggestion
```

File: benchmarks/benchmark.js [hunk 2]
```diff
@@ -162,45 +164,27 @@ function printResults (results) {
...
-                write (chunk, encoding, callback) {
-                  callback()
-                }
-              })
-            )
-            .on('finish', resolve)
-        })
+const experiments = {
+  'http - no keepalive' () {
+    console.log('aaa')
```
Comment (RafaelGSS) [inline benchmarks/benchmark.js:169]:
```suggestion
```

File: benchmarks/benchmark.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ const { WritableStream } = require('stream/web')
 
 const { Pool, Client, fetch, Agent, setGlobalDispatcher } = require('..')
 
-const iterations = (parseInt(process.env.SAMPLES, 10) || 100) + 1
+const iterations = (parseInt(process.env.SAMPLES, 10) || 10) + 1
```
Comment (RafaelGSS) [inline benchmarks/benchmark.js:13]: Why does it need to change? I have run it locally and it's working fine. When `10` it uses only 1 sample.

File: benchmarks/benchmark.js (hunk 1 above)
Comment (mcollina) [reply benchmarks/benchmark.js:132]: This is needed for debugging this and showing that it's using one sample anyway

File: benchmarks/benchmark.js (hunk 3 above)
Comment (mcollina) [reply benchmarks/benchmark.js:13]: Nothing, I committed my current branch.

**Review by ronag — APPROVED** [review]

**Review by ShogunPanda — APPROVED** [review]
LGTM!

**Pull request merged**